docs/modules/operation/pages/admin/geocoder.adoc

```diff
@@ -10,16 +10,69 @@ The geocoder service will only run against nodes that are missing latitute and l
 See the xref:provisioning/directed-discovery.adoc[provisioning] documentation for steps to define asset fields on a requisition.
 
 Resolved coordinates are stored in the database as node asset fields, but the geocoder service does not update requisitions.
-An alternative to the geocoder service is the xref:reference:provisioning/adapters/geoip.adoc[GeoIP provisioning adapter].
-This adapter can look up coordinates based on IP address, and will update the node's requisition definition, bypassing the need for the geocoder service to determine location based on address.
 
-== Configuration
+== Address-based geocoding
 
 To enable or configure the Geocoder Service, use the web-based configuration tool.
 In the web UI, click the menu:Gear Icon[Provisioning > Configure Geocoder Service].
 
 NOTE: If you would prefer to manually edit the config files, you can set the active Geocoder Service via the property `activeGeocoderId` in `$\{OPENNMS_HOME}/etc/org.opennms.features.geocoder.cfg`.
 
+=== Nominatim
+
+For more details, see the link:https://wiki.openstreetmap.org/wiki/Nominatim[official documentation]
+and check the link:https://operations.osmfoundation.org/policies/nominatim/[Nominatim Usage Policy] before using
+the geocoder service.
+
+[options="header"]
+[cols="2,3,1,2"]
+|===
+| Property
+| Description
+| Type
+| Default
+
+4+| *Required*
+
+| acceptUsageTerms
+| To use the Nominatim Geocoder Service you must accept the link:https://operations.osmfoundation.org/policies/nominatim/[Nominatim Usage Policy].
+Set this to `true` to agree to their terms.
+| Boolean
+| false
+
+| url
+| The URL template for the Nominatim Geocoder API.
+The `email` and `query` strings are substituted before making the request.
+| String
+| \https://nominatim.openstreetmap.org/search?format=json&amp;email=\{email}&limit=1&q=\{query}
```

Comment:
This is showing up as a valid URL, not escaped.

Reply:
As is here, this is what I see when rendering locally:
![image](https://user-images.githubusercontent.com/26419758/194148404-012b43fe-dd2f-4551-bbfd-3d5cdfd98577.png)

Reply:
LGTM!